java-maven/maven-perforce/pom.xml

```diff
@@ -0,0 +1,147 @@
+            </plugin>
+
+            <!-- scm plugin configuration -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-scm-plugin</artifactId>
+                <dependencies>
+                    <dependency>
+                        <groupId>com.perforce.p4maven</groupId>
+                        <artifactId>p4maven-provider</artifactId>
+                        <version>${p4maven.version}</version>
+                    </dependency>
+                </dependencies>
+                <configuration>
+                    <!-- we'll be using the 'connection' configuration -->
+                    <connectionType>connection</connectionType>
+                </configuration>
+            </plugin>
+
+            <!-- Use Jar plugin to add Rookout's Perforce changelist and depot path -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>2.1</version>
+                <configuration>
+                    <archive>
+                        <manifest>
+                            <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+                        </manifest>
+                        <manifestEntries>                            
+                            <Rookout-Revision>${buildNumber}</Rookout-Revision>
+                            <Rookout-Repository>Perforce:${depotPath}</Rookout-Repository>
+                        </manifestEntries>
+                    </archive>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <version>2.4.1</version>
+                <configuration>
+
+                    <!-- get all project dependencies -->
+                    <descriptorRefs>
+                        <descriptorRef>jar-with-dependencies</descriptorRef>
+                    </descriptorRefs>
+
+                    <!-- MainClass in mainfest make a executable jar -->
+                    <archive>
+                        <manifest>
+                            <mainClass>com.rookout.HelloWorld</mainClass>
+                        </manifest>
+                    </archive>
+
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>make-assembly</id>
+                        <!-- bind to the packaging phase -->
+                        <phase>package</phase>
+                        <goals>
+                            <goal>single</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+        <resources>
+            <resource>
+                <directory>${basedir}/src/main/java</directory>
```

Comment:
It this source packaging? Why is this here? :)

Reply:
Yes, didn't you read the Readme? :)

Reply:
Maybe add a comment here as well marking it is optional if source tagging is used?

Reply:
Done